marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/ModelController.java

```diff
@@ -264,13 +264,26 @@ private ModelsService newService() {
         public String description;
         public String namespace;
         public String namespacePrefix;
+        public HubCentral hubCentral;
```

Comment:
This class name should be fine for now, but you may later want to make it "HubCentralConfig" just so it's not the same as the existing-and-very-important HubCentral class.